docs/content/error/ng/areq.ngdoc

```diff
@@ -9,3 +9,32 @@ assertion expects is defined and matches the type mentioned in the error.
 
 If the type is `undefined`, make sure any newly added controllers/directives/services are properly
 defined and included in the script(s) loaded by your page.
+
+You can get this error if you accidentally miss to include your AngularJS Controller / Directive / Services file inside your ng-app HTML page.
```

Comment:
Yeah Its exactly same as above text.

But, I was going through some of the accessibility programs and realizes that examples and more contextual meanings reach more than plain text.

 If not needed we can wipe out the new line as you said.  But, I feel examples will improve the meaning of the page.